main.nf

```diff
@@ -494,8 +495,8 @@ process ericscript {
         file(reference) from reference.ericscript
 
     output:
-        set val(sample), file("./tmp/${sample}_ericscript.tsv") optional true into ericscript_fusions
-        file("./tmp/fusions.results.total.tsv") optional true into ericscript_output
+        set val(sample), file("${sample}_ericscript.tsv") optional true into ericscript_fusions
+        set val(sample), file("${sample}_ericscript_total.tsv") optional true into ericscript_output
```

Comment:
Missing `tmp` folder here.

Reply:
Actually no, because I'm moving the file one folder up ;-)